src/flint/fmpq.jl

```diff
@@ -101,6 +101,9 @@ end
 
 zero(a::FlintRationalField) = fmpq(0)
 
+# Exists only to support Julia functionality (no guarantees)
```

Comment:
"No guarantees" means it might be removed at any time?

Reply:
Well, if it causes problems, yes of course. But what I really meant that there are no guarantees that all relevant Julia functionality is supported.